HappyCamps/src/app/guards/auth.guard.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router, } from '@angular/router';
+import { Observable } from 'rxjs';
+import { AuthService } from '../services/auth.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthGuard implements CanActivate {
+  constructor(private auth:AuthService,private router:Router){
+
+  }
+  canActivate():boolean{
+    if(this.auth.isLoggedIn())
```

Comment:
should this have also a condition for the roles or will that be another guard?

Reply:
This is just to check if the user is logged. Not related to user role(this will be verified in the next commit.)